### PR TITLE
Entry point omap UI

### PR DESCRIPTION
**Pull request by Knight7561**

### CODE REVIEW

**Comment by bherr2** [comment]
ASCT+B tab should open first

File: projects/v2/src/app/components/organ-table-selector/organ-table-selector.component.ts [hunk 1]
```diff
@@ -72,10 +107,14 @@ export class OrganTableSelectorComponent implements OnInit {
         });
       });
     });
-    this.hasSomeOrgans = this.selection.selected.length > 0;
+    // this.hasSomeOrgans = this.selection.selected.length > 0;
```
Comment (bherr2) [inline projects/v2/src/app/components/organ-table-selector/organ-table-selector.component.ts:110]: * Remove commented out code
* Remove unnecessary console.log

File: projects/v2/src/assets/omap-sheet-config.json [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+        "bp": 40,
+        "sheetId": "",
+        "gid": "",
+        "url": "https://hubmapconsortium.github.io/ccf-releases/v1.1/docs/omap/omap.html"
+      },
+      {
+        "viewValue": "v1.1",
+        "hraVersion": "v1.2",
+        "value": "Lymph-Node-v1.1",
+        "csvUrl": "",
```
Comment (bherr2) [inline projects/v2/src/assets/omap-sheet-config.json:26]: no URL?

File: projects/v2/src/assets/omap-sheet-config.json [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+    "name": "Intestines",
+    "display": "Intestines",
+    "tissuePreservationMethod": "Fresh Frozen",
+    "imagingMethod": "CODEX",
+    "version": [
+      {
+        "viewValue": "v1.0",
+        "hraVersion": "v1.2",
+        "value": "Intestines-v1.0",
+        "csvUrl": "",
```
Comment (bherr2) [inline projects/v2/src/assets/omap-sheet-config.json:61]: no URL? Does this even work????

File: projects/v2/src/assets/omap-sheet-config.json [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+        "bp": 12,
+        "sheetId": "",
+        "gid": "",
+        "url": "https://hubmapconsortium.github.io/ccf-releases/v1.2/docs/omap/omap-pancreas-codex.html"
+      },
+      {
+        "viewValue": "v1.1",
+        "hraVersion": "v1.3",
+        "value": "Pancreas-v1.1",
+        "csvUrl": "https://hubmapconsortium.github.io/ccf-releases/v1.3/omap/omap-6-pancreas-codex.csvUrl",
```
Comment (bherr2) [inline projects/v2/src/assets/omap-sheet-config.json:214]: bad url

File: projects/v2/src/assets/omap-sheet-config.json [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+        "bp": 30,
+        "url": "https://hubmapconsortium.github.io/ccf-releases/v1.2/docs/omap/omap-lung-celldive.html",
+        "sheetId": "",
+        "gid": ""
+      },
+      {
+        "viewValue": "v1.1",
+        "hraVersion": "v1.3",
+        "value": "Lung-v1.1",
+        "csvUrl": "https://hubmapconsortium.github.io/ccf-releases/v1.3/omap/omap-7-lung-cell-dive.csvUrl",
```
Comment (bherr2) [inline projects/v2/src/assets/omap-sheet-config.json:249]: bad url

File: projects/v2/src/app/components/organ-table-selector/organ-table-selector.component.ts (hunk 1 above)
Comment (Knight7561) [reply projects/v2/src/app/components/organ-table-selector/organ-table-selector.component.ts]: Removed and all urls have been put in place.

File: asctb-api/src/routes/csv.ts [hunk 6]
```diff
@@ -15,7 +15,6 @@ export function setupCSVRoutes(app: Express): void {
    * Fetch a CSV given a link and parse it into json or graph output
    */
   app.get('/v2/csv', async (req: Request, res: Response) => {
-    console.log(`${req.protocol}://${req.headers.host}${req.originalUrl}`);
```
[the remark this answers is not in the document]
Comment (Knight7561) [reply asctb-api/src/routes/csv.ts]: Completed

**Pull request merged**